src/views/TopToolbar/TopToolbar.tsx

```diff
@@ -36,7 +39,12 @@ const TopToolbar: React.FC<Props> = () => {
         </a>
       </BarSegment>
       <BarSegment>
-        <MenuOutlined onClick={togglePanel} style={{ fontSize: 25, cursor: 'pointer' }} />
+        <ConfigurationDialog style={{ ...iconStyle, margin: '0 0.5em' }} />
+        {rightPanelActive ? (
```

Comment:
I like the concept of what you're doing here, however I'd remove it from this PR for two reasons:
1. It's out of the scope of this PR, so if you think it should be added open a different PR. PRs can be small!
2. This doesn't work with the rightPanelActiveMobile property which is the inverse of the rightPanelActive property, meaning the way you have this coded the icon will be backwards on mobile.

Reply:
OK - I'll pull it for now.

Reply:
Done